src/lib/holocene/api-pagination.svelte

```diff
@@ -175,11 +200,27 @@
 <slot name="header" visibleItems={$store.visibleItems} />
 <div class="relative mb-8 flex flex-col gap-4">
   <div class="flex flex-col items-center justify-between gap-4 lg:flex-row">
-    <div class="flex items-center gap-1 lg:gap-2 xl:gap-3">
-      <slot name="action-top-left" visibleItems={$store.visibleItems} />
-    </div>
+    {#if $$slots['action-top-left']}
+      <div class="flex shrink-0 items-center gap-1 lg:gap-2 xl:gap-3">
+        <slot name="action-top-left" visibleItems={$store.visibleItems} />
+      </div>
+    {/if}
+    {#if filterable && filterInputLabel}
+      <Input
+        id="api-pagination-search-input"
```

Comment:
Do we want a search icon here, similar to our other "search" inputs?
```suggestion
        icon="search"
        id="api-pagination-search-input"
```